src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -427,108 +525,24 @@ export class VmNetworkTab extends React.Component<VmNetworkTabProps, VmNetworkTa
                         </Flex>
                     );
                 },
-                props: { width: 10 },
-                hidden: ifaces.every(i => !i.type),
-            },
-            {
-                name: _("Model type"),
-                value: (network, networkId) => {
-                    return (
-                        <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} id={`${id}-network-${networkId}-model`}>
-                            <FlexItem>{network.model}</FlexItem>
-                            {needsShutdownIfaceModel(vm, network) && <NeedsShutdownTooltip iconId={`${id}-network-${networkId}-model-tooltip`} tooltipId="tip-network" />}
-                        </Flex>
-                    );
-                },
-                props: { width: 10 },
-                hidden: ifaces.every(i => !i.model),
             },
             {
                 name: _("MAC address"),
                 value: 'mac',
```

Comment:
IMHO this is a waste of a column -- nobody remembers these. This could move into the details?

Reply:
Right, now I understand -- without a guest agent, we simply can't tell from outside what the names are.